src/components/cylc/SVGTask.vue

```diff
@@ -29,6 +29,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
       held: task.isHeld,
       queued: task.isQueued && !task.isHeld,
       runahead: task.isRunahead && !(task.isHeld || task.isQueued),
+      retry: task.isRetry,
+      wallclock: task.isWallclock,
+      xtriggers: task.isXtriggered,
```

Comment:
We can only display one modifier at a time. This is why the above entries have `&& !` logic, this will need to be extended to cover the new modifiers.

See the Cylc Tui implementation for the display priority:

https://github.com/cylc/cylc-flow/blob/4ad6a23566e617a8dae17265c5d007bbc298aabe/cylc/flow/tui/util.py#L117-L128

Reply:
Dealt with